src/pages/OrgRegister.tsx

```diff
@@ -1,91 +1,117 @@
+/* eslint-disable */
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import Logo from '../assets/logo.svg';
-import { IoPerson, IoMail, IoCreate } from 'react-icons/io5'
+import { FaRegUser, FaRegEnvelope, FaPen } from 'react-icons/fa';
+import { useForm } from 'react-hook-form';
 
 export default function OrgRegister() {
   const [name, setName] = useState("")
 
   const handleChange = (value: string) => {
     setName(value)
-    
   }
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset
+  } = useForm();
```

Comment:
To solve the errors that you are getting when using the error object, give the type of the object as `any` since there is no defined types for the `react-hook-form`
The sample code look like this

```js
 const {
    register,
    handleSubmit,
    formState: { errors },
    reset,
  }: any = useForm();
  
  ```

Then replace the hard-coded error messages.

Reply:
cool